rmf_task_msgs/msg/Phase.msg

```diff
@@ -0,0 +1,24 @@
+# level of the phase tree, from 1, 2....n
+int32 tree_level
```

Comment:
As far as I can figure, I think each `tree_level` value should only appear once in the `Phase[] phases` of `TaskSummary`. After all, what would it mean to have two different `phase` elements with a `tree_level` of `0`? All of the phases for level 0 should be captured in a single `Phase` message.

What we could do instead is have the index in `phases[index]` indicate the phase level. So `phase[0]` is the highest level set of phase descriptions, then `phase[1]` is the next level of subphases, and so on. Then we can get rid of this `tree_level` field.